### PR TITLE
Removed Boston Javascript link since Slack link is broken

### PR DESCRIPTION
**Pull request by VividhPandey003**
Signed-off-by: Vividh Pandey <vividh.pandey.13@gmail.com>

# Remove a Slack Group

- [ Boston Javascript] Name of Slack group 
- [ The invite link that was provided for this slack group no longer works.] Reason for requesting removal

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,6 +42,46 @@ If you would like to add to the list, please see our [contribution guidelines](.
...
+    - [🇳🇬 Nigeria](#-nigeria)
+    - [🇵🇭 Philippines](#-philippines)
+    - [🇵🇱 Poland](#-poland)
+    - [🇷🇴 Romania](#-romania)
+    - [🇿🇦 South Africa](#-south-africa)
+    - [🇷🇸 Serbia](#-serbia)
+    - [🇬🇧 UK](#-uk)
+    - [🇺🇾 Uruguay](#-uruguay)
+    - [🇺🇦 Ukraine](#-ukraine)
+    - [🇻🇳 Vietnam](#-vietnam)
```
Comment (jdwilkin4) [inline README.md:84]: You added a lot of extra duplicate categories. 
Please remove lines 45-84

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (VividhPandey003) [reply README.md:84]: Oh, Didn't realize that, Sorry

**Review by jdwilkin4 — APPROVED** [review]
Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

**Comment by VividhPandey003** [comment]
> Congrats on your first contribution to our repository. 🎉 We look forward to future contributions. 👍

Thanks a lot @jdwilkin4. Would love to contribute more. Do let me know if you require any help somewhere. I am an explorer right now. I am a bit into Web Development but just HTML,CSS,Javascript. I do am well versed with Git and Github. Thank you for giving me your time😄

**Pull request merged**